chapter-make-read.lua

```diff
@@ -157,7 +157,18 @@ local function read_chapter_table()
         local h, m, s, t, n, l
         local thin_space = string.char(0xE2, 0x80, 0x89)
         local line = line:gsub(thin_space, " ")
-        if line:match("^%d+:%d+:%d+") ~= nil then
+        if line:match("^%d+:[0-9,%.]+[,%s]") ~= nil then
```

Comment:
Don't put this match at the very beginning, move down to `elseif line:match("^CHAPTER%d+=%d+:%d+:%d+") ~= nil then` before.
Change to: `if line:match("^%d+:%d+[,%.]?%d+[,%s].*") ~= nil then`